internal/kms/keyprotect.go

```diff
@@ -92,6 +96,14 @@ func initKeyProtectKMS(args ProviderInitArgs) (EncryptionKMS, error) {
 	if err != nil {
 		return nil, err
 	}
+	err = setConfigString(&kms.baseURL, args.Config, keyProtectServiceBaseURL)
+	if errors.Is(err, errConfigOptionInvalid) {
+		return nil, err
+	}
+	err = setConfigString(&kms.tokenURL, args.Config, keyProtectServiceTokenURL)
+	if errors.Is(err, errConfigOptionInvalid) {
+		return nil, err
+	}
```

Comment:
```suggestion
	err = setConfigString(&kms.baseURL, args.Config, keyProtectServiceBaseURL)
	if errors.Is(err, errConfigOptionInvalid) {
		return nil, err
	}else if errors.Is(err,errConfigOptionMissing){
		kms.baseURL = kp.DefaultBaseURL
	}
	err = setConfigString(&kms.tokenURL, args.Config, keyProtectServiceTokenURL)
	if errors.Is(err, errConfigOptionInvalid) {
		return nil, err
	}else if errors.Is(err,errConfigOptionMissing){
		kms.tokenURL = kp.DefaultTokenURL
	}
```
maybe set default here instead of in another function ?

Reply:
@Rakshith-R can be this way too, Done